utils_cv/detection/bbox.py

```diff
@@ -25,14 +30,66 @@ def __init__(self, left: int, top: int, right: int, bottom: int):
         self.standardize()
 
     @classmethod
-    def from_array(cls, arr: List[int]) -> "Bbox":
+    def from_array(cls, arr: List[int]) -> "_Bbox":
         """ Create a Bbox object from an array [left, top, right, bottom] """
         return _Bbox(arr[0], arr[1], arr[2], arr[3])
 
     @classmethod
-    def from_array_xywh(cls, arr: List[int]) -> "Bbox":
-        """ create a Bbox object from an array [left, top, width, height] """
-        return _Bbox(arr[0], arr[1], arr[0] + arr[2], arr[1] + arr[3])
+    def from_array_xywh(cls, arr: List[int]) -> "_Bbox":
```

Comment:
There are a lot new helper functions in here. If they are indeed used, then that's ok. If they are not used, then please delete.

Reply:
Deleted.

Reply:
Thanks for deleting most of the new helper functions. This looks much nicer now and is also easier to maintain going forward